packages/core/src/Parser.ts

```diff
@@ -282,13 +229,12 @@ export class Parser {
         const node = currentContext.node;
         const parentVNode = currentContext.parentVNode;
         const format = currentContext.format[0];
-        const text = Parser.removeFormatSpace(node);
-        for (let i = 0; i < text.length; i++) {
-            const char = text.charAt(i);
-            const parsedVNode = new CharNode(char, { ...format });
-            VDocumentMap.set(parsedVNode, node, i);
+        const parsedVNodes = this.parseNode(node) as CharNode[];
```

Comment:
@dmo-odoo That's right but I figured it would be better to let `CharNode` handle the instantiation. I think it's good practice to keep that as centralized as possible.